src/components/featured-item/featured-item.scss

```diff
@@ -29,9 +30,10 @@ $featured-item: null !default;
 
 .ecl-featured-item__title {
   color: map.get($featured-item, 'title', 'color');
-  font: map.get($featured-item, 'title', 'font-mobile');
   font-weight: map.get($featured-item, 'title', 'font-weight');
   max-width: map.get($featured-item, 'max-width');
+
+  @include responsive-font(map.get($featured-item, 'title', 'font'));
```

Comment:
I see a possible issue here: the font is defined after the font weight, so it will override it.
We may consider changing the order, or having the font weight handled by the mixin

Reply:
thanks romain, i included also the font-weight... also rewrote some stuff because the mixin was not getting the mobile properly.